addons/dex/deifnitions/dex-connectors-def.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: core.oam.dev/v1beta1
```

Comment:
Addon can identify a .cue. So it might be better to keep the .cue file here.

Reply:
For addons, all definition is `YAML` file in the directory.
If we want to change it, it's better that we change it all.